tests/ui

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -eux
+
+echo "==> Install LXD"
+install_lxd
+
+echo "==> Setup node"
+curl -fsSL https://deb.nodesource.com/setup_20.x > setup_node_repo.sh
```

Comment:
can we use node from ubuntu?

https://packages.ubuntu.com/noble/nodejs

Reply:
Node18 is deprecated in playwright which we use for test execution.

Reply:
ack ok

Reply:
why dont we use https://snapcraft.io/node ?

Reply:
Aha! We can probably use it. Will test it locally and replace if it works.

Reply:
yes, that is simpler. using the node snap now.